Fix bugs introduced by setup-sslmate

king-alexander opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->
I introduced a few bugs to the project when I rebased #31. This small pull request fixes those mistakes.
## 💭 Motivation and context ##

<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such -->
<!-- as "closes" or "resolves" to auto-close them on merge. -->

These changes are required to remove my bugs from `develop`.

## 🧪 Testing ##

<!-- How did you test your changes? How could someone else test this PR? -->
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->
I corrected the inadvertent changes from rebasing #31.
<!--
## 📷 Screenshots (if appropriate) ##

Uncomment this section if a screenshot is needed.

-->

## ✅ Pre-approval checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).

## ✅ Pre-merge checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- These boxes should remain unchecked until the pull request has been approved. -->

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

<!-- Remove any of the following that do not apply. -->

- [ ] Add a tag or create a release.

king-alexander committed:
Revert version number

I overwrote Admiral's version number when rebasing #31. This commit
restores the correct version number.

king-alexander committed:
Uncomment dependency names

I reintroduced the commented out dependency names --  managed by the
skeleton-python-library -- when rebasing #31. This commit fixes that
mistake.

mcdonnnj reviewed (changes requested):
Please fix these badges to refer to this repository:

https://github.com/cisagov/admiral/blob/9dd0abe5f80c08e77b47a50f8be60830add11cf3/README.md?plain=1#L3-L6

king-alexander committed:
Make status badges refer to the admiral repository

When I rebased #31, the status badges reverted to referencing the
skelton-python-library repository. This commit changes the references
back to the admiral repository.

dav3r reviewed (approved):
👍

mcdonnnj commented on src/admiral/_version.py:
```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this module."""
-__version__ = "0.2.0"
+__version__ = "1.1.0"
```
Just a reminder to bump the minor version.

king-alexander replied:
[6d98461](https://github.com/cisagov/admiral/pull/39/commits/6d98461ba2412e0cc43f5428b63a688231faaa0c)

king-alexander committed:
Bump version from 1.1.0 to 1.2.0
